app/views/contactPreference/EmailPreferenceView.scala.html

```diff
@@ -24,7 +24,7 @@
 @(form: Form[YesNo])(implicit user: User[_], messages: Messages, appConfig: config.AppConfig)
 
 @questionContent = {
-  <div class="panel">
+  <div class="panel panel-border-wide">
```

Comment:
Changed to make the thickness of the lefthand-side grey bar consistent across all contact pref pages